AnnoyancesFilter/sections/annoyances.txt

```diff
@@ -1284,6 +1284,7 @@ ixbt.com#?#.b-footer > div.b-block > div.b-content--wrapper > div[class]:has(> d
 vchehle.ua##.subscription-block
 pik.ru##.Emails
 officelife.media##a[href="https://t.me/officelife_media"]
+probier-club.de#?#body > div.page-wrapper > section.section:has(> div.row > div.col-md-12 > div.call-to-action > div.row > div.col-md-8 > p:contains(Newsletter))
```

Comment:
It's better to change it to the rule like this `probier-club.de#?#.page-wrapper > section:has(> div.row div.text-center > div.row > div[class] > form[id^="subscribe"])`